src/libraries/System.Globalization.Extensions/tests/IdnMapping/Data/Unicode_9_0/Unicode_9_0_IdnaTest.cs

```diff
@@ -26,7 +26,14 @@ public Unicode_9_0_IdnaTest(string line, int lineNumber)
 
             Type = ConvertStringToType(split[0].Trim());
             Source = EscapedToLiteralString(split[1], lineNumber);
-            bool validDomainName = (split.Length != 5 || split[4].Trim() != "NV8");
+            bool validDomainName = (split.Length != 5 || split[4].Trim() != "NV8"); 
+            
+            // On Server 2019, uses ICU 61.0 which IDNA not supporting the following cases. Ignore these entries there.
+            if (PlatformDetection.IsWindowsServer2019 && Source.EndsWith("\ud802\udf8b\u3002\udb40\udd0a", StringComparison.Ordinal))
+            {
+                Source = "";
```

Comment:
Why not use `Assert.Skip`?

Reply:
Because it doesn't exist? :smile:

Reply:
I was [convinced it does](https://github.com/xunit/xunit/issues/2073#issuecomment-786846718). Then again I regularly have to switch between xUnit and NUnit and I could have just assumed it must exist...

Reply:
It very well might in xunit v3.  We use v2 (v3 isn't released yet to my knowledge).

Reply:
Oh well, I will check back in a year or so, it would be perfect fit here.